addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -94,7 +90,11 @@ def _domain_holiday_status_id(self):
     validation_type = fields.Selection(string='Validation Type', related='holiday_status_id.allocation_validation_type', readonly=True)
     can_reset = fields.Boolean('Can reset', compute='_compute_can_reset')
     can_approve = fields.Boolean('Can Approve', compute='_compute_can_approve')
-    type_request_unit = fields.Selection(related='holiday_status_id.request_unit', readonly=True)
+    type_request_unit = fields.Selection([
+        ('hour', 'Hours'), ('hours', 'Hours'),
+        ('half_day', 'Half Day'),
+        ('day', 'Day'), ('days', 'Day'),
```

Comment:
Useless selections. `added_value_type` is now also without `s`
```suggestion
        ('hour', 'Hours'),
        ('half_day', 'Half Day'),
        ('day', 'Day'),
```